modules/ROOT/pages/install-openshift.adoc

```diff
@@ -142,6 +142,74 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ----
 
+. Apply the following RoleBinding template to name rtf an any additional namespace:
```

Comment:
see a couple of typos:
How about this one: 
`Apply the following RoleBinding template to rtf namespace and any additional namespaces:`